inc/Engine/CriticalPath/CriticalCSSSubscriber.php

```diff
@@ -575,20 +633,78 @@ public function exclude_inline_js( array $excluded_inline ) {
 	/**
 	 * Defer loading of CSS files.
 	 *
-	 * @since  3.6.2 Uses the AsyncCSS.
 	 * @since  2.10
 	 *
-	 * @param string $html HTML code.
+	 * @param string $buffer HTML code.
 	 *
 	 * @return string Updated HTML code
 	 */
-	public function async_css( $html ) {
-		$instance = AsyncCSS::from_html( $this->critical_css, $this->options, $html );
-		if ( ! $instance instanceof AsyncCSS ) {
-			return $html;
+	public function async_css( $buffer ) {
+		if ( rocket_get_constant( 'DONOTROCKETOPTIMIZE' ) ) {
+			return;
+		}
```

Comment:
Should this be `return $buffer;`? Not sure.

Reply:
yes it should you're right https://github.com/wp-media/wp-rocket/pull/2895/commits/9317abe8d97f163c02fffe768baf885e561b6d2e